packages/esm-patient-orders-app/src/order-basket/order-basket.workspace.tsx

```diff
@@ -116,21 +124,27 @@ const OrderBasket: React.FC<DefaultWorkspaceProps> = ({
             <InlineNotification
               kind="error"
               title={t('errorCreatingAnEncounter', 'Error when creating an encounter')}
-              subtitle={t('tryReopeningTheWorkspaceAgain', 'Please try launching the workspace again')}
+              subtitle={creatingEncounterError}
               lowContrast={true}
               className={styles.inlineNotification}
               inline
```

Comment:
[InlineNotification](https://react.carbondesignsystem.com/?path=/docs/components-notifications-inline--overview) does not accept a prop called `inline`.